rosidl_typesupport_c/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ endif()
 
 # Default to C++14
 if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
Is this necessary? If so, we should update the comment above.

Reply:
Undone in https://github.com/ros2/rosidl_typesupport/pull/127/commits/fb0258fcd7466695c2c50db4f87492198a36d220